runbooks/README.md

```diff
@@ -2,26 +2,19 @@
 Table of Contents
 =================
 
-* [How to Detach BFD Server Instances for Development or Debugging](./how-to-detach-bfd-server-instances-for-development-or-debugging.md#how-to-detach-bfd-server-instances-for-development-or-debugging)
-   * [Scale Out to Avoid Potential Performance Degradation](./how-to-detach-bfd-server-instances-for-development-or-debugging.md#scale-out-to-avoid-potential-performance-degradation)
-   * [Detach the Instance](./how-to-detach-bfd-server-instances-for-development-or-debugging.md#detach-the-instance)
-   * [Scale In After Detaching an Instance](./how-to-detach-bfd-server-instances-for-development-or-debugging.md#scale-in-after-detaching-an-instance)
-   * [Patching Regularly When Instances Endure Beyond ~1 Day](./how-to-detach-bfd-server-instances-for-development-or-debugging.md#patching-regularly-when-instances-endure-beyond-1-day)
-   * [Terminating the Instance](./how-to-detach-bfd-server-instances-for-development-or-debugging.md#terminating-the-instance)
-
-* [How to Mark SNYK Findings as Ignored](./how-to-mark-snyk-findings-ignored.md#how-to-mark-snyk-findings-as-ignored)
+* [Recreating server-regression QuickSight Dashboards](./how-to-create-server-regression-dashboards.md#recreating-server-regression-quicksight-dashboards)
```

Comment:
Whatever github actions keeps doing this doesnt seem to know how it wants this formatted.